modules/effects/src/main/scala/dev/profunktor/redis4cats/redis.scala

```diff
@@ -1383,6 +1383,25 @@ private[redis4cats] class BaseRedis[F[_]: Concurrent: ContextShift: Log, K, V](
 
   override def scriptFlush: F[Unit] =
     async.flatMap(c => F.delay(c.scriptFlush())).futureLift.void
+
+  /** ***************************** HyperLoglog API **********************************/
+  override def pfAdd(key: K, values: V*): F[Long] =
+    async
+      .flatMap(c => F.delay(c.pfadd(key, values: _*)))
+      .futureLift
+      .map(Long.unbox)
```

Comment:
I think we usually want the opposite: `Long.box`

Reply:
that's what I initially thought, but all the other methods in here seem to use `Long.unbox`

Reply:
actually, some use `unbox` and some use `box`, so I'll swap to `box` since it makes more sense

Reply:
never mind, it doesn't compile with `box`

Reply:
hmmm I might have missed the `unbox` usage in other places, will need to revisit that at some point.